style.css

```
@@ -1831,40 +1831,64 @@ h1, h2, h3, h4, h5, h6, .mission-modern-title, .feature-title h2, #join .section
-    border-radius: 20px;
-    box-shadow: 0 8px 32px rgba(0,0,0,0.18);
-    padding: 3.5rem 4rem;
+    border-radius: 16px;
+    box-shadow: 0 12px 40px rgba(0,0,0,0.15);
+    padding: 2.75rem 3.25rem;
     max-width: 99vw;
     width: 100%;
-    min-width: 320px;
-    max-width: 480px;
+    min-width: 300px;
+    max-width: 420px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

max-width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)